src/main/antora/modules/ROOT/pages/jdbc/template.adoc

```diff
@@ -0,0 +1,48 @@
+[[jdbc.template]]
+= Template API
+
+Spring Data JDBC offers the javadoc:org.springframework.data.jdbc.core.JdbcAggregateTemplate[] as a more direct means to load and persist entities in a relational database.
+Repositories use to a large extend the `JdbcAggregateTemplate` to implement their features.
+
+This section will highlight only the most interesting parts of the `JdbcAggregateTemplate`.
+For a more complete overview see the JavaDoc of `JdbcAggregateTemplate`.
+
+== Accessing the JdbcAggregateTemplate
+
+`JdbcAggregateTemplate` is intended to be used as Spring bean.
+If you have set up your application including Spring Data JDBC you may configure a dependency on `JdbcAggregateTemplate` in any Spring bean and the Spring Framework will inject a properly configured instance.
+
+This includes fragments you use to implement custom methods for your Spring Data Repositories, allowing you to use `JdbcAggregateTemplate` to customize and extend your repositories.
+
+== Persisting
+
+`JdbcAggregateTemplate offers three types of methods for persisting entities: `save`, `insert`, and `update`.
+Each comes in two flavors:
+Operating on single aggregates, named exactly as mentioned above, and with a `All` suffix operation on an `Iterable`.
+
+`save` does the same as the method of same name in a repository.
+
+`insert` and `update` skip the test if the entity is new and assume a new or existing aggregate as indicated by their name.
+
+== Querying
+
+`JdbcAggregateTemplate` offers a considerable array of methods for querying aggregates and about collections of aggregates.
+There is one type of method that requires special attention.
+That's the methods taking a `Query` as an argument.
+They allow the execution of programmatically constructed queries as shown in the following example.
```

Comment:
Change to:

They allow the execution of programmatically constructed queries, as follows: